packages/Breadcrumbs/stories/Breadcrumbs.mdx

```diff
@@ -0,0 +1,6 @@
+import { Props, Description } from "@storybook/addon-docs/blocks";
```

Comment:
Sorry I mean this file the documentation should be more than the readme file, if you can explain how to use this component would be great for everybody,  there are more people using our library and documentation is starting to be more important each day :)

Reply:
Added! resolve the conversation